src/smart-components/role/add-role-permissions/add-role-permissions-summary-content.js

```diff
@@ -38,21 +45,34 @@ const AddRolePermissionSummaryContent = ({ selectedPermissions = [], role = {} }
             </Text>
           </GridItem>
           <GridItem span={9}>
-            <Text component={TextVariants.p}>
+            <TextContent component={TextVariants.p}>
               {selectedPermissions.map((permission, index) => (
                 <Text key={index}> {permission.uuid} </Text>
               ))}
-            </Text>
+            </TextContent>
           </GridItem>
+          {hasCostResources && (
+            <>
```

Comment:
Just a nit pick, but can you use `Fragment` instead of `<>` I tend to overlook this.